test/lint.js

```diff
@@ -8,4 +8,12 @@ var blint = require("blint");
   });
 });
 
+["src"].forEach(function(dir) {
+  blint.checkDir(dir, {
+    browser: true,
+    blob: "// CodeMirror, copyright (c) by Marijn Haverbeke and others\n// Distributed under an MIT license: http:\/\/codemirror.net\/LICENSE\n\n",
```

Comment:
I think it's okay that way. It's a separate commit, so we can easily take it out again, though.